packages/cli/src/commands/proxy/index.ts

```diff
@@ -0,0 +1,91 @@
+import * as http from 'node:http';
+
+import chalk from 'chalk';
+import type { CommandModule } from 'yargs';
+
+import { consoleLog } from '../../utils.js';
+
+import { createOidcResponseHandler, createProxy, isLogtoRequestPath } from './utils.js';
+
+const proxy: CommandModule<
+  unknown,
+  {
+    url?: string;
+    tenant?: string;
+    port: number;
+    endpoint?: string;
+  }
+> = {
+  command: ['proxy'],
+  describe: 'Command for Logto proxy',
+  builder: (yargs) =>
+    yargs
+      .options({
+        url: {
+          alias: ['u', 'sign-in-experience-url'],
+          describe: 'The URL of your custom sign-in experience page',
+          type: 'string',
+        },
```

Comment:
i would choose `--experience-uri` and `-e`

Reply:
I think `-e` is taken by the global argument `--env`, should I override it within the `proxy` cli command?

Reply:
Discussed offline and use `--experience-uri` and `-x` instead